tests/tacacs/utils.py

```diff
@@ -338,3 +376,20 @@ def log_exist(duthost):
 
     exist = wait_until(timeout, 1, 0, log_exist, duthost)
     pytest_assert(exist, "Not found aaa config update log: {}".format(command))
+
+
+def print_tacacs_creds(tacacs_creds):
```

Comment:
>print_tacacs_creds [](http://example.com/codeflow?start=4&length=18)

This function is not used, and seems leaking test credential. Do you really need it?

Reply:
Fixed, removed.